Dockerfile

```diff
@@ -1,9 +1,9 @@
 # Runs master branch from github of lando/lando-messaging
 # Requires a lando config file mounted at /etc/lando_config.yml.
 FROM python:2.7.13
-RUN pip install --upgrade git+git://github.com/Duke-GCB/lando-messaging.git
 ADD . /src
 WORKDIR /src
+RUN pip install --upgrade git+git://github.com/Duke-GCB/lando-messaging.git
```

Comment:
I didn't want to add a module that wasn't useful for others to pypi.
I think with the way python tooling works you end up needed to add your modules to pypi even if they aren't useful in a general way.